src/system-health/health_checker/hardware_checker.py

```diff
@@ -35,27 +36,34 @@ def _check_asic_status(self, config):
         if config.ignore_devices and 'asic' in config.ignore_devices:
             return
 
-        temperature = self._db.get(self._db.STATE_DB, HardwareChecker.ASIC_TEMPERATURE_KEY, 'temperature')
-        temperature_threshold = self._db.get(self._db.STATE_DB, HardwareChecker.ASIC_TEMPERATURE_KEY, 'high_threshold')
-        if not temperature:
-            self.set_object_not_ok('ASIC', 'ASIC', 'Failed to get ASIC temperature')
-        elif not temperature_threshold:
-            self.set_object_not_ok('ASIC', 'ASIC', 'Failed to get ASIC temperature threshold')
-        else:
-            try:
-                temperature = float(temperature)
-                temperature_threshold = float(temperature_threshold)
-                if temperature > temperature_threshold:
-                    self.set_object_not_ok('ASIC', 'ASIC',
-                                           'ASIC temperature is too hot, temperature={}, threshold={}'.format(
-                                               temperature,
-                                               temperature_threshold))
-                else:
-                    self.set_object_ok('ASIC', 'ASIC')
-            except ValueError as e:
-                self.set_object_not_ok('ASIC', 'ASIC',
-                                       'Invalid ASIC temperature data, temperature={}, threshold={}'.format(temperature,
-                                                                                                            temperature_threshold))
+        ASIC_TEMPERATURE_KEY_LIST = self._db.keys(self._db.STATE_DB,
+                                                  HardwareChecker.ASIC_TEMPERATURE_KEY + '*')
```

Comment:
@Junchao-Mellanox , We can modify it here like 'TEMPERATURE_INFO|*ASIC*'. Currently, it does like 'TEMPERATURE_INFO|ASIC*'. Do you want to change the it like *ASIC*?.

Reply:
I feel that we can have it like 'TEMPERATURE_INFO| ASIC *'.  ASIC Key name must be starting with "ASIC ..." and accordingly, it can be changed in API 2.0 Thermal list.